simulator/func_sim/alu.h

```diff
@@ -8,6 +8,7 @@
 #define ALU_H
 
 #include "trap_types.h"
+#include "multiplication.h"
```

Comment:
I suggest to include that header to "riscv_instr.cpp" directly.

Reply:
Unfortunately, it causes ``not declared in this scope`` errors in ``alu.h``